src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static Task[] tasks=new Task[100];;
+    private static int taskCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        welcomeMessage();
+        Scanner read=new Scanner(System.in);
+        String command=read.nextLine();
+        while(!command.equals("bye")){
+            printDividingLine();
+            switch(command) {
+            case "list":
+                listTasks();
+                break;
+            default:
+                if(command.startsWith("done ")){
+                    doneTask(command);
+                }else if (command.startsWith("todo ")){
+                    addToDo(command);
+                }else if (command.startsWith("deadline ")){
+                    addDeadline(command);
+                }else if (command.startsWith("event ")){
+                    addEvent(command);
+                }else{
+                    System.out.println("Please give the type of the task.");
+                }
+            }
```

Comment:
Would it be more readable if the default statement only has one case? Maybe the if-else within the default could be implemented within the switch statement?